pom.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.tempoplugin</groupId>
+  <artifactId>tempo-events-example</artifactId>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <name>System Test Plugin: Tempo Events Example</name>
+  <description>JIRA Plugin example to listen to Tempo Events</description>
+  <packaging>atlassian-plugin</packaging>
+
+  <organization>
+    <name>Tempo</name>
+    <url>http://www.tempo.io</url>
+  </organization>
+
+  <properties>
+    <atlassian.spring.scanner.version>1.2.13</atlassian.spring.scanner.version>
+    <!-- This key is used to keep the consistency between the key in atlassian-plugin.xml and the key to generate bundle. -->
+    <atlassian.plugin.key>${project.groupId}.${project.artifactId}</atlassian.plugin.key>
+    <!-- TestKit version 6.x for JIRA 6.x -->
+    <testkit.version>6.3.11</testkit.version>
```

Comment:
remove or use a version that is compatible with the Jira version you are declaring (7.1.0)